imageregistry/v1/types.go

```diff
@@ -337,6 +337,18 @@ type AzureNetworkAccess struct {
 }
 
 type AzureNetworkAccessInternal struct {
+	// networkResourceGroupName is the resource group name where the cluster's vnet
+	// and subnet are. When omitted, the registry operator will use the cluster
```

Comment:
The resource group for the vnet and subnet must always be the same I assume?

Reply:
Yes, updated comment.